mozregression/cli.py

```diff
@@ -550,7 +557,15 @@ def validate(self):
 
         user_defined_bits = options.bits is not None
         options.bits = parse_bits(options.bits or mozinfo.bits)
-        fetch_config = create_config(options.app, mozinfo.os, options.bits, mozinfo.processor)
+        if options.arch is not None:
+            self.logger.info("app is %s" % options.app)
+            if options.app != "gve":
+                self.logger.info("arch option being ignored for non-GVE app.")
```

Comment:
Let's make this a warning

```suggestion
                self.logger.warning("--arch ignored for non-GVE app.")
```